.github/workflows/build-book.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       # Test for bad links and ensure alt tags for usability
       - name: Check HTML using htmlproofer
-        uses: chabad360/htmlproofer@master
+        uses: chabad360/htmlproofer@v5.0.9
```

Comment:
```suggestion
        uses: chabad360/htmlproofer@v2
```

Reply:
this might work ? the action is not specifically synced with htmlproofer itself i think? 

https://github.com/marketplace/actions/htmlproofer